Components/FlightControl/RocketSM.cpp

```diff
@@ -428,6 +428,15 @@ RocketState Ignition::HandleCommand(Command& cm)
             // This is a transition directly to ARM (no checks required)
             nextStateID = RS_ARM;
             break;
+        case RSC_IGINITION_CMD:
+            SOAR_PRINT("This was for testing purposes...\n");
+            break;
+        case RSC_MANUAL_IGNITION_CONFIRMED:
+            TimerTransitions::Inst().ignitionConformation = true;
```

Comment:
I'd prefer to make this a function called `ConfirmManualIgnition()` just for clarity, but doesn't really matter

Reply:
Would it be ok to leave it because i think that the ENUM Name already conveys that information and it avoids having to process another command in the TimerTransitions task